conda/recipes/libcucim/conda_build_config.yaml

```diff
@@ -18,16 +15,3 @@ c_stdlib_version:
 
 cmake_version:
   - ">=3.30.4"
-
-# The CTK libraries below are missing from the conda-forge::cudatoolkit package
-# for CUDA 11. The "*_host_*" version specifiers correspond to `11.8` packages
-# and the "*_run_*" version specifiers correspond to `11.x` packages.
-
-cuda11_libcufile_host_version:
-  - "1.4.0.31"
-
-cuda11_libcufile_run_version:
-  - ">=1.0.0.82,<=1.4.0.31"
-
-cuda11_libnvjpeg_host_version:
-  - "11.6.0.55"
```

Comment:
These are CUDA 11 pinnings for dependencies outside the Conda `cudatoolkit` package

None of that is relevant for CUDA 12+, which has the full CTK as packages with associated pinning

Also the usage of these values was dropped already with PR: https://github.com/rapidsai/cucim/pull/889

Hence these can just be dropped